### ISSUE TITLE
Ingest all Hausumringe datasets for whole of bavaria

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#283]
## Description
See Epic #145 
For now, we are only using Oberfranken, but we want all other regions as well

## Acceptance criteria
* All datasets ingested
* Sources documented
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
